### DISCUSSION TITLE
I want the 3D view from the old Firefox DevTools back

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#147] [Wants]
It’s impossible to see when an element is floating right (or left) outside of the document. The 3D view feature was that killer feature to debug this, but it’s gone… RIP.

<hr>

https://webwewant.fyi/wants/30/

**Comment by ststimac** [comment]
This is currently available in Edge / Chromium DevTools now. 

https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/3d-view/
